# generated by patch-package 6.4.7 on 2021-12-17 15:16:05
#
# command:
#   npx patch-package eslint-plugin-jsx-a11y
#
# declared package:
#   eslint-plugin-jsx-a11y: 6.5.1
#
diff --git a/node_modules/eslint-plugin-jsx-a11y/lib/index.js b/node_modules/eslint-plugin-jsx-a11y/lib/index.js
index d809156..a4e3457 100644
--- a/node_modules/eslint-plugin-jsx-a11y/lib/index.js
+++ b/node_modules/eslint-plugin-jsx-a11y/lib/index.js
@@ -12,7 +12,6 @@ module.exports = {
     'aria-proptypes': require('./rules/aria-proptypes'),
     'aria-role': require('./rules/aria-role'),
     'aria-unsupported-elements': require('./rules/aria-unsupported-elements'),
-    'autocomplete-valid': require('./rules/autocomplete-valid'),
     'click-events-have-key-events': require('./rules/click-events-have-key-events'),
     'control-has-associated-label': require('./rules/control-has-associated-label'),
     'heading-has-content': require('./rules/heading-has-content'),
